dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -138,11 +141,13 @@ def _add_helper_merge_profile_histograms(self, other1, other2):
         for i, method in enumerate(self.histogram_bin_method_names):
             combined_values = other1._histogram_to_array(
                 method) + other2._histogram_to_array(method)
-            bin_counts, bin_edges = self._get_histogram(
+            bin_counts, bin_edges, suggested_bin_count = self._get_histogram(
                 combined_values, method)
             self.histogram_methods[method]['histogram']['bin_counts'] = \
                 bin_counts
             self.histogram_methods[method]['histogram']['bin_edges'] = bin_edges
+            self.histogram_methods[method]['suggested_bin_count'] = \
```

Comment:
Do we really need a "suggested_bin_count"?

What is the advantage of that over being explicit and when is the suggestion ignored.

Reply:
Suggested occurs bc most algorithms will suggest bins < 1k. This allows us to reconstruct the suggested bins at report time from the 1k bin histogram

Reply:
The next refactor would allow us to need create only 1 histogram at profile time. The other histograms could be recreated from the one.